scripts/all_year_days

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+#
+# Launch a command every day of current year
+#
+# Depends on: faketime (`apt install faketime` i.e.)
+#
+# EXAMPLE:
+# ./scripts/all_year_days pipenv run pytest -vvs --no-cov --disable-warnings tests/api/test_loans_rest.py
+#
+
+# variables
+launch_command="$@"
+
+# functions
+error+exit() {
+  echo -e "$1" && exit 1
+}
+
+# tests
+if [[ -z "${launch_command}" ]]; then
+  echo "No command given." && exit 1
+fi
+
+if [[ -z $(which faketime) ]]; then
+  echo "faketime command is missing! Please install libfaketime." && exit 1
+fi
+
+# initialization
+current_year=`date +"%Y"`
+start_date="${current_year}-01-01"
+# display all days of year (from current year)
+for i in {0..366}; do
+  date="$(date -u +"%Y-%m-%d" -d "${start_date} +$i days") 9:30:42"
```

Comment:
@BadrAly : I changed the way to process date. Should be OK now!
Tested this morning (~2H35mn) with success!

Reply:
``` 
date: illegal time format
usage: date [-jnRu] [-d dst] [-r seconds] [-t west] [-v[+|-]val[ymwdHMS]] ...
            [-f fmt date | [[[mm]dd]HH]MM[[cc]yy][.ss]] [+format]
#### CHOOSEN DATE:  9:30:42 ####
Running specified command failed: Permission denied
'tests/api/test_loans_rest.py' FAILED this day:  9:30:42.

Reply:
I thing you have to run the script with `sudo`